src/main/app/src/app/shared/edit/edit-behandelaar/edit-behandelaar.component.html

```diff
@@ -5,11 +5,11 @@
 <div class="static-text" (click)="edit(true)">
```

Comment:
Misschien is het ook een idee om de [class.readonly]="readonly" op het hoogste niveau te zetten en de onderdelen te verbergen met css

Reply:
er zit geen overerving in de templates, de styling zit al op het hoogste niveau; het niet tonen van de shadow voor het bewerken in read-only modus